includes/html/api_functions.inc.php

```diff
@@ -1019,15 +1019,28 @@ function get_port_info(Illuminate\Http\Request $request)
 
 function search_ports(Illuminate\Http\Request $request)
 {
+    $field = $request->route('field');
     $search = $request->route('search');
-    $value = "%$search%";
+
+    if (isset($search)) {
+        $value = "%$search%";
+    } else {
+        $value = "%$field%";
+    }
+
     $ports = Port::hasAccess(Auth::user())
```

Comment:
and rename this variable to $query to avoid confusion.

Reply:
changed